docs/blog/2017-02-21-1-0-progress-update-where-came-from-where-going/index.bkup

```diff
@@ -1,139 +0,0 @@
----
-title: "Gatsby 1.0 update"
-date: "2017-02-23"
-author: "Kyle Mathews"
----
-
-*hint — the solution isn't reverting to an idealized "simpler" way of
```

Comment:
This is still a great article... maybe I should polish it up and publish it 🤔

Reply:
It just needs an updated title: _Gatsby ~1.0~ 2.0 update_ :)